index.bs

```diff
@@ -1796,20 +1796,19 @@ elem.addEventListener('dragover', (e) => {
   // Prevent navigation.
   e.preventDefault();
 });
+
 elem.addEventListener('drop', async (e) => {
-  // Prevent navigation.
   e.preventDefault();
 
-  // Process all of the items.
-  for (const item of e.dataTransfer.items) {
-    // kind will be 'file' for file/directory entries.
-    if (item.kind === 'file') {
-      const entry = await item.getAsFileSystemHandle();
-      if (entry.kind === 'file') {
-        handleFileEntry(entry);
-      } else if (entry.kind === 'directory') {
-        handleDirectoryEntry(entry);
-      }
+  const fileHandlesPromises = [...e.dataTransfer.items]
+    .filter((item) => item.kind === 'file')
+    .map((item) => item.getAsFileSystemHandle());
```

Comment:
you don't need () around item when there is just one argument. ```.map(item => item.getAsFileSystemHandle());``` works fine

Reply:
It's a style guide thing. Google highly [recommends](https://google.github.io/styleguide/jsguide.html#:~:text=tip%3A%20always%20using%20parentheses%20even%20for%20single-parameter%20arrow%20functions%20can%20avoid%20situations%20where%20adding%20parameters%2C%20but%20forgetting%20to%20add%20parentheses%2C%20may%20result%20in%20parseable%20code%20which%20no%20longer%20works%20as%20intended.) adding braces.

Reply:
I see. I usually leave them out as they just add more noise and make the code less readable in my opinion :-)

Reply:
This is the spec, the Google Style Guide doesn't apply here. Feel free to remove, I don't have hard feelings. It's a prettier rule that I have in muscle memory :-)